build.sbt

```diff
@@ -33,8 +33,7 @@ val versions = new {
   val scala                = "2.12.20" // Must use Scala 2.12.x for sbt plugins
   val airframe             = "24.12.2"
   val sonatypeZapperClient = "1.3"
-  val sttp                 = "4.0.0-M16"
-  val sonatypeClient       = "0.3.0"
+  val sonatypeClient       = "0.5.0"
```

Comment:
I guess anything that's built on JDK 8 would be a progress, but if each pull request takes two months to land, we might as well try to fix the timeout issue in one go?